website/docs/language/functions/cidrcontains.mdx

```diff
@@ -0,0 +1,28 @@
+---
+page_title: cidrcontains - Functions - Configuration Language
+description: |-
+  The cidrcontains function determines whether a given IP address or an address prefix given in CIDR notation is within a given IP network address prefix.
+---
+
+# `cidrcontains` Function
+
+`cidrcontains` determines whether a given IP address or an address prefix given in CIDR notation is within a given IP network address prefix.
+
+Note that both arguments must be of the same type, either IPv4 or IPv6. A family mismatch will result in an error.
```

Comment:
```suggestion
Note that both arguments must be of the same address family, either IPv4 or IPv6. A family mismatch will result in an error.
```

Tiny Tiny Tiny Nitpick: The only reason I suggest this is that I don't want it to be ambigious about what `type` means (ie, here someone could think that they both have to be a CIDR address, rather than ipv4 vs ipv6)

Reply:
Fair enough. Done.